### ISSUE TITLE
Should be a property to disable the diff view

### ISSUE THREAD

**Issue by jonespm** [w8tcha/CKEditor-AutoSave-Plugin#63]
Even though the diffview should be fixed after #27 it would be nice to have a configuration option to just skip the diff view just import whatever content has been autosaved directly after clicking okay. 

I think the dialog could just say

An auto-saved version of this content from "{0}" has been found. Would you like to import the autosaved content overwriting your current content?
